plutarch-docs/Concepts/GenericProgramming.md

```diff
@@ -56,57 +75,35 @@ The code of a datatype.
 This is a list of lists of its components. The outer list contains one element per constructor. The inner list contains one element per constructor argument (field).
 ```
 
+```haskell
 data Tree = Leaf Int | Node Tree Tree
+```
 
 is supposed to have the following `Code`:
 
 ```haskell
-type instance Code (Tree a) =
-'[ '[ Int ]
-, '[ Tree, Tree ]
-]
+type TreeCode =
+  '[ '[ Int ]
+   , '[ Tree, Tree ]
+   ]
```

Comment:
I think the mention of `Code` should also be removed from the preceding line if we're to avoid introducing the `Code` associated type in the codeblock.

Reply:
I have changed this to `Representation`?